src/web/routes/application/postcode/postcode.js

```diff
@@ -1,3 +1,29 @@
+const request = require('request-promise')
+const { wrapError } = require('../common/formatters')
+const { transformOsPlacesApiResponse } = require('./adapters')
+
+const OS_PLACES_API_URL = 'https://api.ordnancesurvey.co.uk/places/v1/addresses/postcode'
+
+const behaviourForPost = (config) => async (req, res, next) => {
+  const { OS_PLACES_API_KEY } = config.environment
+  const { postcode } = req.body
+
+  try {
+    const addressLookupResults = await request({
+      uri: `${OS_PLACES_API_URL}?postcode=${postcode}&key=${OS_PLACES_API_KEY}`,
```

Comment:
This will need updating as the uri doesn't include any paths. Currently it's set to `https://api.ordnancesurvey.co.uk`

Reply:
I might leave adding the URL to config for a separate PR

Reply:
I've updated this with value from config